examples/cogs/basic_not_extension.py

```diff
@@ -0,0 +1,58 @@
+# In this example you will learn how to make cogs (not extensions)
+
+import discord
+from discord.ext import commands
+from discord.ext.commands import HelpCommand
+from discord import app_commands
+
+from typing import Optional, Type, Any
+
+
+class MyCog(commands.Cog):
+    def __init__(self, bot) -> None:
+        self.bot = bot # Assign bot who is setuping cog
+    
+    @commands.Cog.listener()
+    async def on_message(self, message: discord.Message):
+        if message.author.bot:
+            return
+        if message.content.startswith('!ping'):
+            await message.reply('Pong!')
+
+    @commands.bot_has_permissions(administrator=True)
+    @commands.command()
+    async def admin_check(self, ctx: commands.Context):
+        await ctx.send('You are admin!')
+
+# Create bot with cogs
+class Bot(commands.Bot):
+    def __init__(
```

Comment:
Unnecessary...

Reply:
I personally like having `_init_` there so you I can use the type hinting for the when I initialized some variables

Reply:
> Unnecessary to override the base class `__init__`

Maybe remove this method?